prov/rxm/src/rxm_rma.c

```diff
@@ -131,15 +131,21 @@ rxm_ep_format_rma_res_lightweight(struct rxm_ep *rxm_ep, uint64_t flags,
 
 static inline ssize_t
 rxm_ep_format_rma_buf(struct rxm_ep *rxm_ep, size_t total_size,
+		      const struct fi_msg_rma *orig_msg,
 		      struct rxm_rma_buf **rma_buf, struct rxm_tx_entry *tx_entry)
 {
+	size_t i;
+
 	*rma_buf = rxm_rma_buf_get(rxm_ep);
 	if (OFI_UNLIKELY(!*rma_buf))
 		return -FI_EAGAIN;
 
 	tx_entry->state = RXM_TX_RMA;
 	tx_entry->rma_buf = *rma_buf;
 	(*rma_buf)->pkt.hdr.size = total_size;
+	(*rma_buf)->rxm_iov.count = orig_msg->iov_count;
+	for (i = 0; i < orig_msg->iov_count; i++)
+		(*rma_buf)->rxm_iov.iov[i] = orig_msg->msg_iov[i];
```

Comment:
Does this commit need to be back-ported to 1.6?

Reply:
Yes, this commit should be there (#3983)